src/pages/Assess.vue

```diff
@@ -6,13 +6,16 @@
         <span v-if="fields.pages.length < 20">Page {{pageIdx}}/{{fields.pages.length}}</span>
         <span v-else>{{percentDone}}%</span>
         <v-stepper-items>
-          <v-stepper-content
+          <v-stepper-content 
             v-for="(page, idx) in fields.pages"
             :key="page.id"
-            :step="idx + 1">
+            :step="idx + 1"
+            class="assessment-page"
+            :id="page.id"
```

Comment:
does this, the id tag, need to be an integer? If so, I think it should probably be in a different attribute (e.g. `data-idx`). If not, it should probably have a prefix (e.g. `page-3`)